greenhouse_management/models.py

```diff
@@ -55,3 +55,18 @@ class CropTypes(models.TextChoices):
                                     # on_delete=models.CASCADE, <-- to consider
                                     # related_name="default_greenhouses") 
     # environments = models.ManyToManyField(Environment, related_name="greenhouses")
+    
+    
+class Enviroment(models.Model):
+    
+    date = models.DateTimeField()
+    temperature = models.DecimalField(max_digits=5, decimal_places=2)  
+    air_humidity = models.DecimalField(max_digits=5, decimal_places=2)  
+    light_level = models.DecimalField(max_digits=5, decimal_places=2)  
+    PAR = models.DecimalField(max_digits=5, decimal_places=2)  
```

Comment:
nazwy pól powinny składać się z małych liter

Reply:
spokojnie na_razie wrzuciłem żeby nie stracić tego w razie co bo różnie z kompem bywa, to jeszcze nieskończone jest

Reply:
@Pyame to zobacz czy mozesz jako draft oznaczyc